multiversx_sdk_network_providers/transaction_events.py

```diff
@@ -28,13 +28,21 @@ def from_http_response(response: Dict[str, Any]) -> 'TransactionEvent':
 
         return result
 
+    def to_dictionary(self) -> Dict[str, Any]:
+        return {
+            "address": self.address.bech32(),
+            "identifier": self.identifier,
+            "topics": [item.__str__() for item in self.topics],
```

Comment:
Some topics are not utf8 strings, if I remember well. Perhaps return the `hex` representation here?

Reply:
Done!